[OSDOCS#17160] using-ruby CQA (#117479)
openshift/openshift-docs: modules/templates-rails-creating-database-service.adoc

[id="templates-rails-creating-database-service_{context}"]
= Creating the database service

[role="_abstract"]

> You can run `oc new-app postgresql` to create a database service for your Rails application. Setting environment variables for the database name, username, and password ensures the service connects correctly to your Rails application.

Reviewer: ```suggestion
You must create a database service for your Rails application. Be sure to set the environment variables for the database name, username, and password. These are required for the service to connect correctly to your Rails application.
```

You can change the values of these environment variables to any values you choose. The variables are as follows:

* `POSTGRESQL_DATABASE`
* `POSTGRESQL_USER`
* `POSTGRESQL_PASSWORD`

> Setting these variables ensures:

Reviewer: ```suggestion
Setting these variables ensures that the following occurs:
```
